kubernetes/config/kube_config_model.h

```diff
@@ -7,113 +7,113 @@
+    KUBECONFIG_PROPERTY_TYPE_CLUSTER,
+    KUBECONFIG_PROPERTY_TYPE_USER,
+    KUBECONFIG_PROPERTY_TYPE_USER_EXEC,
+    KUBECONFIG_PROPERTY_TYPE_USER_AUTH_PROVIDER
+} kubeconfig_property_type_t;
 
-    typedef struct kubeconfig_property_t {
-        kubeconfig_property_type_t type;
-        char *name;
-        union {
-            struct {            /* context */
-                char *cluster;
-                char *namespace;
-                char *user;
-            };
-            struct {            /* cluster */
-                char *server;
-                char *certificate_authority_data;
-            };
-            struct {            /* user */
-                char *token;
-                char *client_certificate_data;
-                char *client_key_data;
-                struct kubeconfig_property_t *auth_provider;
-                struct kubeconfig_property_t *exec;
-                int insecure_skip_tls_verify;
-                char *username;
-                char *password;
-            };
-            struct {            /* user exec */
-                char *command;
-                char *apiVersion;
-                keyValuePair_t **envs;
-                int envs_count;
-                char **args;
-                int args_count;
-            };
-            struct {            /* user auth provider */
-                char *access_token;
-                char *client_id;
-                char *client_secret;
-                char *cmd_path;
-                char *expires_on;
-                char *expiry;
-                char *id_token;
-                char *idp_certificate_authority;
-                char *idp_certificate_authority_data;
-                char *idp_issuer_url;
-                char *refresh_token;
-            };
-        };
-    } kubeconfig_property_t;
+typedef struct kubeconfig_property_t {
+    kubeconfig_property_type_t type;
+    char *name;
+    union {
+        struct {            /* context */
+            char *cluster;
+            char *namespace;
+char *user;
+};
+struct {            /* cluster */
+    char *server;
+    char *certificate_authority_data;
+    int insecure_skip_tls_verify;
+};
+struct {            /* user */
+    char *token;
+    char *client_certificate_data;
+    char *client_key_data;
+    struct kubeconfig_property_t *auth_provider;
+    struct kubeconfig_property_t *exec;
+    char *username;
+    char *password;
+};
+struct {            /* user exec */
+    char *command;
+    char *apiVersion;
+    keyValuePair_t **envs;
+    int envs_count;
+    char **args;
+    int args_count;
+};
+struct {            /* user auth provider */
+    char *access_token;
+    char *client_id;
+    char *client_secret;
+    char *cmd_path;
+    char *expires_on;
+    char *expiry;
+    char *id_token;
+    char *idp_certificate_authority;
+    char *idp_certificate_authority_data;
+    char *idp_issuer_url;
+    char *refresh_token;
+};
+};
```

Comment:
Could you please restore the indentation as before ?
Because I find your IDE cannot recognize the nested structure.